scripts/scil_volume_distance_map.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Compute distance map between two binary masks using KDTree. The distance map
+is the Euclidean distance from each voxel of the first mask to the closest
+voxel of the second mask.
+It's worst case scenario is two very big masks that are far appart or a very
```

Comment:
"Corner cases include two very big masks that are far apart or one mask inside another."

Reply:
They are not corner cases, they are slowest case scenario, I changed the phrase accordingly